packages/web/components/Player/Section/PlayerHero.tsx

```diff
@@ -52,7 +55,7 @@ export const PlayerHero: React.FC<HeroProps> = ({ player, editing }) => {
       withoutBG
       p={[4, 2]}
     >
-      {isOwnProfile && !editing && (
+      {isOwnProfile && isMainNet && !editing && (
```

Comment:
It seems like you're hiding the edit button, but not letting the player know they have to connect.

I need to look at the test instance to better understand this flow…

Reply:
@dysbulic It was due to [this issue ](https://github.com/MetaFam/TheGame/issues/1223) ?

Reply:
As mentioned above, the best behavior would be to show the edit button, but prompt the user to change their chain if they're not on mainnet when they click it.